magefiles/utils_test.go

```diff
@@ -78,6 +78,10 @@ func (m *QuayClientMock) CreateRobotAccount(organization string, robotName strin
 	return nil, nil
 }
 
+func (m *QuayClientMock) GetRobotAccount(organization string, robotName string) (*quay.RobotAccount, error) {
```

Comment:
Unit test was complaining about absence of `GetRobotAccount` routine definition. I think since we are using `QuayClientMock` struct to implement `QuayService` interface, so this dummy routine is needed.